uswds/uswds-compile
# USWDS-Compile - Dependencies: Use node LTS 20

# Summary

Updating to node LTS 20 & NPM dependencies.

## Related issue

Closes #71. _Related to uswds/uswds#5010._

## Problem statement

Updating to stay up-to-date with latest Node LTS and dependencies. As well as avoiding compatibility issues with CloudGov pages once node 16 is deprecated.

## Testing and review

1. Go to Sandbox branch `test-uswds-compile-78-node-update`.
2. Run `npm install`.
3. Run tasks listed in `gulpfile.js`.
4. All tasks should work as expected.
